src/models/p2sh_signature.ts

```diff
@@ -57,6 +57,7 @@ class P2SHSignature {
     const signatures: Record<number, string> = {};
     for (const sig of arr.slice(1)) {
       const parts = sig.split(':');
+      // eslint-disable-next-line prefer-destructuring -- Destructuring would make this harder to read
```

Comment:
Maybe we should use destructuring

Reply:
Done on https://github.com/HathorNetwork/hathor-wallet-lib/pull/679/commits/de3e7750df496b6c9575101ae2049de29f652e55